src/cpl/mct/rtm_cpl_indices.F90

```diff
@@ -31,10 +31,6 @@ module rtm_cpl_indices
 
   integer, public :: nflds_x2r = 0
 
-  !TODO - nt_rtm and rtm_tracers need to be removed and set by access to the index array
```

Comment:
I'm not completely sure why this was moved to another module. Could you explain that?

Reply:
There could be no references to mct or nuopc cap code from within rtm. rtm_cpl_indices is ONLY used by the mct code and therefore there cannot be references to that when using the nuopc cap.